src/Ractive.js

```diff
@@ -59,6 +59,7 @@ defineProperties( Ractive, {
 	Promise:        { value: Promise },
 
 	// support
+	enhance:        { writable: true, value: false },
```

Comment:
I quite like it... makes me think of Blade Runner

![6lre7ol](https://cloud.githubusercontent.com/assets/1162160/10622395/462d84c2-7755-11e5-8309-109e77cb0466.gif)

The *real* reason I like it is because it ties it to its stated purpose. It's also nice and easy to type! But it's certainly up for debate